tests/osaka/eip7692_eof_v1/eip6206_jumpf/test_nonreturning_validation.py

```diff
@@ -17,23 +17,57 @@
 
 
 @pytest.mark.parametrize(
-    "code_section",
+    "code",
     [
-        pytest.param(Section.Code(Op.STOP, code_outputs=0), id="stop"),
-        pytest.param(Section.Code(Op.INVALID, code_outputs=0), id="invalid0"),
-        pytest.param(
-            Section.Code(Op.ADDRESS + Op.POP + Op.INVALID, code_outputs=0), id="invalid1"
-        ),
-        pytest.param(Section.Code(Op.RETURN(0, 0), code_outputs=0), id="return"),
-        pytest.param(Section.Code(Op.RETF, code_outputs=0), id="retf0"),
-        pytest.param(Section.Code(Op.PUSH0 + Op.RETF, code_outputs=1), id="retf1"),
+        Op.STOP,
+        Op.INVALID,
+        Op.ADDRESS + Op.POP + Op.INVALID,
+        Op.RETURN(0, 0),
+        Op.RETF,
+        Op.PUSH0 + Op.RETF,
```

Comment:
I think we should retain the `pytest.param(..., id` , because it currently works ok for single opcodes: `code_STOP`, `code_INVALID`, but on composite opcodes it shows an index: `code_0`, `code_1`, etc.